sf-bitcoin-meetup/2017-09-04-jonas-schenlli-bip150-bip151.md

```diff
@@ -1,19 +1,19 @@
 ---
 title: Jonas Schenlli Bip150 Bip151 (2017-09-04)
```

Comment:
can you fix up the file name and the title field?

Reply:
just a note that file name changes, also change the url, breaking any old shared links.